### DISCUSSION TITLE
Where can I lay my hand on current/recent documentation (ie for the stable version 1.10)

### DISCUSSION THREAD

**Discussion by ondohotola** [ledgersmb/LedgerSMB#7443] [Q&A] [answered]
As mentioned in #7435 I am contemplating migration. 

The [demo version](https://demo.cloud.efficito.com/erp/1.5/login.pl) is 1.5 and I am looking for some specific features (in the first instance) not in the particular order listed below

1. Importing Invoices (I generate CSVs from a Web Portal, other formats such as JSON would not a problem)
2. VAT (I generate submission forms for the local Receiver for each period via `bash` script, currently via `SELECT` statements to the backend.
   1. Tax Collected (AR)
   2. Tax Paid (AP)
   3. A combination of both or exporting of the above for example as CSV
   `Tax Reports` raise an error on the demo
5. Importing (Bank-) Transactions (OFX or API)
6. I assume I'll have to rewrite my (invoice.tex) templates.  I see from the [invoice template](https://github.com/ledgersmb/LedgerSMB/blob/093e54352987f28180bc3dfee6547eb996707a12/templates/demo/invoice.tex) that a different templating is used. Can I just replace the templating (ie `<%` with `<?lsmb` and `%>` with `?> ` or do I have to rewrite a standard template (not a drama)?

If it's more convenient to raise each item as an individual Discussion (issue), please let me know.

**Comment by christian-eriksson** [comment, marked as answer]
I think the [book](https://book.ledgersmb.org/) is the closest there is to current documentation. Though I should say that getting in contact directly with the team in the [Matrix Channel](https://app.element.io/#/room/#ledgersmb:matrix.org) is recommended, it's a very helpful community :)

You might have already found the [project website](https://ledgersmb.org/) where some [older docs](https://ledgersmb.org/content/documentation) are hosted, I assume so as you are asking for current docs.
